packages/grafana-ui/src/components/RefreshPicker/RefreshPicker.tsx

```diff
@@ -93,7 +100,23 @@ export function intervalsToOptions({ intervals = defaultIntervals }: { intervals
   SelectableValue<string>
 > {
   const intervalsOrDefault = intervals || defaultIntervals;
-  const options = intervalsOrDefault.map((interval) => ({ label: interval, value: interval }));
+  const options = intervalsOrDefault.map((interval) => {
+    const duration = parseDuration(interval);
+    const unit = Object.keys(duration) as Array<keyof Duration>;
+    const amount = duration[unit[0]];
+    let describedIntervalLabel = interval;
+
+    if (amount !== undefined) {
+      const formattedUnit = amount > 1 ? unit : unit[0].slice(0, -1);
+      describedIntervalLabel = amount + ' ' + formattedUnit;
+    }
```

Comment:
Instead, are we able to use `formatDuration` from date-fns? formatDuration

This could be simplified down to 

```typescript
const ariaLabel = formatDuration(parseDuration(interval))
```

Reply:
ooooft I didn't even realise there was a `formatDuration`. that feels like the one, and avoids any manual manipulation of the string.

nice find @joshhunt 🙌

Reply:
@joshhunt it seems `formatDuration` expects duration values to be of type number instead. Had to convert the values to numbers before passing it in to`formatDuration`. Would push my solution right away